docs/workshop.md

```diff
@@ -761,19 +761,23 @@ You have previously created an Azure Function which reacts to cache expiry by li
 
 This new Azure Function will power an API for retrieving a user's browsing history, e.g. which products the users have viewed recently.
 
-But before delving into the Azure Function's code, let's first take a look at the Streams you currently have at disposal.
+![Example of sequence diagram to introduce history-func](./assets/history-func-usage-sequence-diagram.png)

```

Comment:
Explique le diagram et encadre des choses dessus si besoin

Reply:
I added a quick explanation for the sequence diagram without going too much in details as I think it is currently self explanatory